pages/osx/awk.md

```diff
@@ -0,0 +1,24 @@
+# awk
+
+> Find and Replace text within file(s).
+> More information: <https://ss64.com/osx/awk.html>.
+
+- Define the input field separator = regular expression fs:
+
+`awk [ -F fs ]`
+
+- Assign values before prog is executed:
+
+`awk [ -v var=value ]`
+
+- Specify a pattern to match within progfile:
+
+`awk [ 'prog' | -f progfile ]`
```

Comment:
```suggestion
- Execute a specific program.

`awk --file {{path/to/file}}`
```

Almost the same issue as above.
Optional fix: `specific` may sound better.
Mandatory fix: [path format is broken](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths).